source/dotnet/wholesale-api/Infrastructure/EventPublishers/IIntegrationEventDispatcher.cs

```diff
@@ -19,5 +19,5 @@ public interface IIntegrationEventDispatcher
     /// <summary>
     /// Fetches number of integration events and dispatches them (bulk).
     /// </summary>
-    Task<bool> DispatchIntegrationEventsAsync(int numberOfIntegrationEventsToDispatch);
+    Task<bool> BulkDispatchIntegrationEventsAsync(int numberOfMessagesToDispatch);
```

Comment:
To me, "bulk" is an implementation detail of no interest to the caller.
```suggestion
    Task<bool> DispatchIntegrationEventsAsync(int numberOfMessagesToDispatch);
```